docs/users-stats/global.d.ts

```diff
@@ -1512,13 +1512,13 @@ declare namespace Deno {
   export function truncate(name: string, len?: number): Promise<void>;
 
   export interface NetAddr {
-    transport: "tcp" | "udp";
+    transport: 'tcp' | 'udp';
```

Comment:
I know it's just formatting, but do you know  what this file is for? I hadn't encountered it before

Reply:
TBH no idea 🤷‍♂️ 
@ff6347 What about you?